components/admin_console/manage_roles_modal/manage_roles_modal.jsx

```diff
@@ -64,11 +64,11 @@ export default class ManageRolesModal extends React.PureComponent {
         this.state = getStateFromProps(props);
     }
 
-    UNSAFE_componentWillReceiveProps(nextProps) { // eslint-disable-line camelcase
+    componentDidUpdate(prevProps) { // eslint-disable-line camelcase
```

Comment:
Can you also remove the `// eslint-disable-line` comments? They're no longer needed

Reply:
Will do.  Thank you for your feedback